src/ImageSharp/Formats/Jpeg/Components/Decoder/HuffmanTable.cs

```diff
@@ -119,7 +143,8 @@ public HuffmanTable(MemoryAllocator memoryAllocator, ReadOnlySpan<byte> codeLeng
                 }
             }
 
-            Unsafe.CopyBlockUnaligned(ref this.Values[0], ref MemoryMarshal.GetReference(values), 256);
+            this.isDerived = true;
+            return this;
```

Comment:
This will return a copy of the struct, so we can't go fluent here. (And we probably do not want repeated calls for `Derive()`.)

Reply:
Yeah, that'll be gone in the next commit. I forgot to lazy create the fast AC tables also so none of it actually works!

Reply:
Does this mean that something is missing from our coverage?

Reply:
Oh wait, I haven't noticed the failing tests.

It's a sign that I should go to bed now 😄